### PR TITLE
Work and about myself sections

### PR DESCRIPTION
**Pull request by Sanja969**
### Work and AboutMe sections - Milestone 2

In this milestone I have:

- Added work and AboutMe section
- Styled work and about me section
- Changed description in README file
- I stick to the style of **Figma** template 2
- I used images from **Figma** template

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Sanja969 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great following the design
✔️ No linter error
✔️ Professional PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ]  [OPTIONAL] you should always capitalized the commits so please try to do that next time 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -16,38 +16,180 @@
...
+            <ul id="socialMedia">
+                <li class="imgSoc1"><img src="Icons/Icon-GitHub.png" alt="GitHub Icon" width="18" height="20"></li>
+                <li class="imgSoc1"><img src="Icons/Icon-Linkedin.png" alt="LinkedIn Icon" width="18" height="20"></li>
+                <li class="imgSoc1"><img src="Icons/Icon-Angelist.png" alt="Angelist Icon" width="18" height="20"></li>
+                <li class="imgSoc1"><img src="Icons/Icon-Twitter.png" alt="Twitter Icon" width="18" height="20"></li>
+                <li class="imgSoc1"><img src="Icons/Icon-Medium.png" alt="Medium Icon" width="18" height="20"></li>
+            </ul>
+        </section>
+
+        <section id="work">
```
[the remark this answers is not in the document]
Comment (Sanja969) [reply index.html:50]: Thank you for all reviews, but this one I consider too much. Ass you can see I was using a grid display in others cases like was proposed in the lesson. If you would change it to be grid I would have many other settings to change. In this case, flex display is better and easier.

[the remark this answers is not in the document]
Comment (Sanja969) [reply index.html:50]: In the end, I change it, but in this way, I'm not learning anything, I more spending time on changing unnecessary stuff, and then I don't have time for learning and resting.

**Review by xtrahuman — APPROVED** [review]
# Approved ✔️ 
Hi @Sanja969, 

Great job implementing all the changes requested by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**